packages/gatsby-plugin-styled-components/src/gatsby-node.js

```diff
@@ -38,7 +38,7 @@ exports.pluginOptionsSchema = ({ Joi }) =>
         `By default minifiers cannot properly perform dead code elimination on styled components because they are assumed to have side effects. This enables "pure annotations" to tell the compiler that they do not have side effects.`
       ),
     disableVendorPrefixes: Joi.boolean()
-      .default(false)
+      .default(true)
```

Comment:
I don't think we want to change default? Just ensure the new prop is used (with inverse of old one value)?

Reply:
For me, the idea was to match the default of styled-components.
If you want to keep it false happy to change it back.